### PR TITLE
cosi: fix problem in cosi document

### PR DESCRIPTION
**Pull request by Madhu-1**
**Description of your changes:**
Fixed a wording problem and moved the app pod to the default namespace where the secret was created and also removed the init container from the sample as the busybox doesn't have setup-aws-credentials CLI pre-installed.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md [hunk 1]
```diff
@@ -200,13 +199,6 @@ spec:
     volumeMounts:
     - name: cosi-secrets
       mountPath: /data/cosi
-  initContainers:
```
Comment (Madhu-1) [inline Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md:203]: I removed the init container as we don't have `setup-aws-credentials` in busybox container. any alternative image for this one? if not can were remove the annotation approach mentioned in the doc?

Comment (thotz) [reply Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md:203]: Any bash scripting container image will do, I though busybox is light weight container. Idea here is provide a way  to parse the BucketInfo as Enviroment variables

**Review by thotz — COMMENTED** [review]
Apart from changes in the sample application pod, LGTM

File: Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md (hunk 1 above)
Comment (Madhu-1) [reply Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md:203]: @thotz Thanks, am not sure any image will have the `setup-aws-credentials` CLI command in it. if you can specify the exact container image which contains `setup-aws-credentials` will replace it, for now this pod is failing to start with below error

```
Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason     Age              From               Message
  ----     ------     ----             ----               -------
  Normal   Scheduled  7s               default-scheduler  Successfully assigned default/sample-app to minikube
  Normal   Pulling    5s (x2 over 6s)  kubelet            Pulling image "busybox"
  Normal   Pulled     5s               kubelet            Successfully pulled image "busybox" in 1.117956858s (1.117975281s including waiting)
  Normal   Created    4s (x2 over 5s)  kubelet            Created container init-cosi
  Normal   Started    4s (x2 over 5s)  kubelet            Started container init-cosi
  Normal   Pulled     4s               kubelet            Successfully pulled image "busybox" in 1.074652516s (1.074667163s including waiting)
  Warning  BackOff    2s (x2 over 3s)  kubelet            Back-off restarting failed container init-cosi in pod sample-app_default(98bcef58-61a8-4a14-9558-4741780afc9b)
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f cosi $]kubectl logsectl logs po/sample-app
Defaulted container "sample-app" out of: sample-app, init-cosi (init)
Error from server (BadRequest): container "sample-app" in pod "sample-app" is waiting to start: PodInitializing
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f cosi $]kubectl logs po/sample-app init-cosi
/bin/sh: setup-aws-credentials: not found
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f cosi $]
```

Comment (travisn) [reply Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md:203]: Can we mount the secret as environment variables directly? Or is the COSI secret not compatible with that option?

Comment (Madhu-1) [reply Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md:203]: not sure about that @thotz any idea about it?

Comment (thotz) [reply Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md:203]: It is .json format. We need to parse cannot export directly as env like OBC

Comment (travisn) [reply Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md:203]: Up on lines 177-180 we have the example to export the env vars, but if that is in an init container, the variables would only apply to the init container and wouldn't be passed on to the main container. So it seems the app will just need to parse the json.

**Review by BlaineEXE — APPROVED** [review]
Approving with a request to help improve readability of a console command

**Review by BlaineEXE — CHANGES_REQUESTED** [review]
I changed my mind. Please fix the command for readability

**Review by travisn — APPROVED** [review]

**Review by BlaineEXE — APPROVED** [review]

**Pull request merged**